applications/asset_tracker_v2/src/main.c

```diff
@@ -520,7 +506,8 @@ void main(void)
 	struct app_msg_data msg = { 0 };
 
 	if (!IS_ENABLED(CONFIG_LWM2M_CARRIER)) {
-		handle_nrf_modem_lib_init_ret();
+		err = nrf_modem_lib_init(NORMAL_MODE);
+		handle_nrf_modem_lib_init_ret(err);
```

Comment:
Could call `nrf_modem_lib_init` in `handle_nrf_modem_lib_init_ret` to boil this down to a single function call.
`handle_nrf_modem_lib_init_ret` could also be renamed to `modem_init` or something equivalent.

`modem_init` is a little bit more descriptive that makes it clear to the user that calling `nrf_modem_lib_init` involves initializing the modem.

Reply:
Thanks, added. I think the `CONFIG_NRF_MODEM_LIB` guard is needed for `nrf_modem_lib_init` though, for the application to support `native_posix` builds.

Reply:
right! Thanks